app/models/illiad_openurl.rb

```diff
@@ -0,0 +1,35 @@
+###
+#  Class to handle creation of ILLiad OpenURL request
+###
+class IlliadOpenurl
+  def initialize(scan, redirect_url)
+    @scan = scan
+    @redirect_url = redirect_url
+  end
+
+  def illiad_page
+    { 'Action': '10', 'Form': '30', 'rft.genre': 'scananddeliver' }
+  end
+
+  def illiad_item_data
+    { 'rft.location': @scan.origin,
+      'rft.call_number': @scan.holdings.first.callnumber,
+      'rft.item': @scan.holdings.first.barcode }
```

Comment:
Should this actually be `@scan.barcodes.first`? I think that's the barcoded item the user selected on the form (@jkeck, is that right?)